tfhe/src/integer/server_key/radix_parallel/add.rs

```diff
@@ -709,3 +731,58 @@ impl ServerKey {
         reduce_impl(self, ct_seq, op)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::should_hillis_steele_propagation_be_faster;
```

Comment:
are those run properly with the filters we have ?

Reply:
they should on the non fast test